models/users.js

```diff
@@ -56,6 +58,25 @@ const addOrUpdate = async (userData, userId = null) => {
   }
 };
 
+const addJoinData = async (userData) => {
+  try {
+    await db.collection("joining_data").doc(userData.userName).set(userData);
```

Comment:
instead of doing this you can have a variable which contains the value `firestore.collection("joining-data")` and use it here directly that would be much cleaner, one more thing, can we change the name to `intro` from joining data that is better in my opinion